CourseParsing/ParseScript.py

```diff
@@ -31,15 +31,15 @@ def get_course_codes():
                     'MGMT', 'MNS', 'MTHEL', 'ME', 'MTE', 'MEDVL', 'MENN', 'MOHAWK', 'NE', 'OPTOM', 'PHARM',
                     'PHIL', 'PLAN', 'PSCI', 'PORT', 'PD', 'PDARCH', 'PDPHRM', 'REC', 'RS', 'RUSS', 'REES',
                     'SCI', 'SCBUS', 'SMF', 'SDS', 'SVENT', 'SOCWK', 'SWREN', 'STV', 'SOC', 'SPAN', 'SI',
-                    'THPERF', 'UNIV', 'VCULT'}
+                    'THPERF', 'UNIV', 'VCULT']
 
     dbc = DatabaseSender()
 
-    dbc.execute("DROP TABLE IF EXISTS " + dbc.antireqs_table + ";")
-    dbc.execute("DROP TABLE IF EXISTS " + dbc.prereqs_table + ";")
-    dbc.execute("DROP TABLE IF EXISTS " + dbc.communications_table + ";")
-    dbc.execute("DROP TABLE IF EXISTS " + dbc.breadth_table + ";")
-    dbc.execute("DROP TABLE IF EXISTS " + dbc.course_table + ";")
+    dbc.drop_table(dbc.antireqs_table)
```

Comment:
If you are a new member to UWPath, will you run into errors since you had no tables to drop to begin with? I think that's why the IF EXISTS statement exist - unless the flow has changed

Reply:
Oracle does not have IF EXISTS. The drop_table function handles it

Reply:
more specifically, the drop_table function defines a db procedure to check if the table exists and if it does it will execute the drop table command. Since we are dropping all tables at the start of each dataparse, no need to check if the table exists when we recreate them as you mention in your comments below

Reply:
Ok sounds good so oracle does have a safety check

Reply:
Not sure what you mean. Oracle has no built in equivalent to IF EXISTS. drop_table is just a function I wrote to manually check if a table exists. Feel free to take a look